js/scripts/zone_loaders/playground_zone_loaders.js

```diff
@@ -0,0 +1,64 @@
+const loadPlaygroundCaleb = () => {
+    const queueAssets = () => {
+
```

Comment:
WOW. This script is great! Thank you. One thought, we may want to consider breaking apart these constants into their own scripts if they become full and have directories for each dimension depending on the sizes and quantities to keep organization and code clean. @B1GD3VD0G

Reply:
I've been just using VSCode's ability to shrink down functions, and only opening up the one I'm focusing on. I'm open to that, but I don't think it will be necessary. I kinda wanna try and keep the head of index.html from being 4,000 lines too lol.

Reply:
Sounds good. Let's see how it goes. Personally since index will likely be looked at less frequently, I am open to it being fairly large to keep lots of other code cleaner and perhaps we won't have to do that.